ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -675,6 +675,7 @@ tests:
       OPENSHIFT_INSTALL_RELEASE_IMAGE: release:latest
     env:
       ASSISTED_ZTP_CONFIG: |
+        ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE="quay.io/openshift-release-dev/ocp-release:4.11.0-0.nightly-multi-2022-07-19-022323"
```

Comment:
is there anything special about this exact image, or can it be ``4.11.0-rc.6`` as well?
I prefer the more official versions so we can easily advance it afterwards (assuming they are multi-arch and work correctly)

Reply:
There was supposed to be quay.io/openshift-release-dev/ocp-release:4.11.0-rc.6-multi, but it does not exist

Reply:
https://multi.ocp.releases.ci.openshift.org/releasestream/4-stable-multi/release/4.11.0-multi-rc.6

Reply:
Anyway, it's probably being pushed as we speak so let me change it